libs/designer/src/lib/core/state/templates/templateSlice.ts

```diff
@@ -258,7 +312,9 @@ const loadTemplateFromGithub = async (templateName: string, manifest: Template.M
       workflowDefinition: (templateWorkflowDefinition as any)?.default ?? templateWorkflowDefinition,
       manifest: templateManifest,
       workflowName: templateManifest.title,
+      workflowNameValidationError: undefined,
```

Comment:
Why are we storing the validation errors separately for each field in the store? Can we have a top level property called errors and key it with the particular section to get errors from that section?